admin.go

```diff
@@ -87,11 +86,8 @@ func AdminRebuildDBHandler(ctx *handlerContext, w http.ResponseWriter, r *http.R
 
 	//return /*
 	//db.Close()
-	os.Remove("./RebbleAppStore.db")
```

Comment:
This line is important, because if the database already exists, we need to dump its contents first. And if it is corrupted, we have to create a new one.

So I think the best option here is to do
```go
os.Remove("./RebbleAppStore.db")
ctx.db, err := sql.Open("sqlite3", "./RebbleAppStore.db")
```